src/assets/scss/MasonryLayout.scss

```diff
@@ -12,3 +12,16 @@
   box-sizing: border-box;
   min-height: 100vh;
 }
+
+.masonry-layout {
+  column-count: 2;
+  column-gap: 1rem;
+}
+.masonry-layout__panel {
+  break-inside: avoid;
+  padding-bottom: 1rem;
+}
+.masonry-layout__panel-content {
```

Comment:
For cleaner code, I think we should always have one line space before creating a css class